docker/README.md

```diff
@@ -0,0 +1,20 @@
+# Docker support
+
+The files in this directory allow you to build a docker image.  The query coordinator assembly must be 
+copied to `query-coordinator-assembly.jar` in this directory before building.
```

Comment:
We could... I'm not sure I'm in love with the -LATEST suffix since the file is still called latest even when it is no longer the latest, not sure how much value that adds in this case especially since we won't have multiple jar files sitting around.